core/src/main/java/org/opensearch/sql/analysis/ExpressionAnalyzer.java

```diff
@@ -213,6 +214,13 @@ public Expression visitHighlightFunction(HighlightFunction node, AnalysisContext
     return new HighlightExpression(expr);
   }
 
+  @Override
+  public Expression visitPositionFunction(PositionFunction node, AnalysisContext context) {
+    Expression left = node.getLeft().accept(this, context);
```

Comment:
can we used a better description than `left` and `right`?  I'd prefer to use something like `substringExpr` or `stringPatternExpr` and then `searchStringExpr`?

Reply:
Addressed in 9a6b7b5d8962cff8af410638da5151e9ee6a36b9